.github/workflows/release.yml

```diff
@@ -37,60 +67,64 @@ jobs:
             });
             for (const release of releases.data) {
               if (release.draft) {
-                      core.info(`${release}`)
+                      core.info(release)
                       core.exportVariable('RELEASE_ID', release.id)
                       return
               }
             }
             core.setFailed(`Draft release not found`)
 
+      - name: Download all artifacts
+        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3
+        # no name provided, download all artifacts. Puts them in folders.
+
+      - name: Display structure of downloaded files
+        run: ls -R
+
       - name: Upload release assets
+        id: upload_release_assets
```

Comment:
tried several other GHA, but none allowed for uploading files with globs into _existing_ releases, by name and not tag. So found no way for them to be compatible with release-drafter.